x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -254,6 +254,7 @@ export const DETECTION_ENGINE_PREPACKAGED_URL =
   `${DETECTION_ENGINE_RULES_URL}/prepackaged` as const;
 export const DETECTION_ENGINE_PRIVILEGES_URL = `${DETECTION_ENGINE_URL}/privileges` as const;
 export const DETECTION_ENGINE_INDEX_URL = `${DETECTION_ENGINE_URL}/index` as const;
+export const DETECTION_ENGINE_SIGNAL_INDEX_URL = `${DETECTION_ENGINE_URL}/signal/index` as const;
```

Comment:
Should it be internal as well as the other routes added in this PR?
If you think it has to be public, please discuss with @elastic/security-detections-response-alerts team.

Reply:
This should be internal.